tests/System.Memory.Polyfill.Tests/System.Memory.Polyfill.Tests.csproj

```diff
@@ -1,13 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\tools\common.props" />
   <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
     <TargetFrameworks>net46;netcoreapp2.1</TargetFrameworks>
-    <AssemblyOriginatorKeyFile>../../tools/test_key.snk</AssemblyOriginatorKeyFile>
```

Comment:
Why did you remove these properties?

Reply:
@ericstj suggested that they were not necessary for tests.